Add landing page

ellaml opened the pull request:
- Added landing page that displayed the last three reviews
- Added single_preview template for displayed review in the landing page
- Modified views.py to get the last three reviews (by date) and send it
  to the page
- Updated app_layout css (navigation bar)

The "More Reviews" button on the right bottom is redirecting to Reviews Page.

![image](https://user-images.githubusercontent.com/73358484/118246371-8d1f2280-b4aa-11eb-8216-2c062886edad.png)



fix: #107

ellaml committed:
Added Landing Page

- Added landing page that displayed the last three reviews
- Added single_preview template for displayed review in the landing page
- Modified views.py to get the last three reviews (by date) and send it
  to the page
- Upadted app_layout css (navigation bar)

ellaml committed:
Merge branch 'main' into add-landing-page

ellaml committed:
fix missing blank line in css file

ellaml commented:
I'm still having trouble with aligning the search box on the nav-bar (this is why I hate using bootstrap -.-) 
Will leave it to next week with all the other required CSS improvements

VeraTs commented on homepage/static/css/landing.css:
```diff
@@ -0,0 +1,97 @@
+	display: flex;
+	flex-direction: column;
+	align-self: center;
+	text-shadow: 3px 3px 3px #888888;
```
Suggestion: a lighter tone for the text-shadow? Or make the shadow wider, maybe; because when looking at it and trying to read it gets a bit headache-y

( found this fantastic website for hex colors: [colorhexa](https://www.colorhexa.com/888888))

ellaml replied:
will do. :)

ellaml replied:
@VeraTs I changed it, wdyt?

VeraTs replied:
Looks Great!

VeraTs reviewed (commented):
Looks amazing!
Just this little css bit and that's all 😄

oriAdler commented on homepage/views.py:
```diff
@@ -9,7 +9,10 @@ def app_layout(request):
 
 def landing(request):
-    return render(request, 'homepage/landing/landing.html')
+    last_reviews = Review.objects.all().order_by('-date')
```
```suggestion
    last_reviews = Review.objects.all().order_by('-date')[:3]
```

ellaml replied:
Great, I thought [:3] will raise an error when theres less than 3, but python keeps surprising me :)
Changed and moved this logic to a new `landing_page_feed()` function in order to test is separately.

oriAdler reviewed (commented):
Looks really good!
The navbar looks much better, and in general, the landing page is beautiful.
I would suggest adding a small test for the new function ``` landing```

ellaml committed:
CSS fix and adding test for landing feed

- Modified landing css file
- Created landing_page_feed function in Review model
- Added test in tests.py for the new function
- Modified views.py to call the function and just pass its result to
  render

oriAdler reviewed (approved):
LGTM
